irida_sistr_results/sistr_writer.py

```diff
@@ -226,16 +229,18 @@ def write(self, sistr_results):
 
         self._formatting()
 
+        sample_create_msg = ' using only samples created after ' + self.sample_created_min_date.strftime(
+            '%Y-%m-%d %H:%M:%S') if self.sample_created_min_date else ''
         self._write_row([
             "Results generated from " + self.appname + " version=" + __version__ + " connecting to IRIDA=" + self.irida_url + " as user=" + self.username + " on date=" + datetime.now().strftime(
-                '%Y-%m-%d %H:%M:%S')])
+                '%Y-%m-%d %H:%M:%S') + sample_create_msg])
```

Comment:
This looks like ISO 8601 date/time format. You could use `datetime.now().isoformat(sep=' ', timespec='seconds')` (to use the same `space` separator between date and time and only report up to seconds).

I noticed that this line is added to the bottom of the CSV and the bottom of the first worksheet in the XLSX. If it's not too much trouble, would it be possible to change this so that this line is a comment in the CSV (i.e. prefixed with `#`) before the header? And have this info put onto a different sheet in the XLSX (like with `staramr` and maybe with other bits of useful info)?

Reply:
Thanks. I fixed all dates I print out in ae4936c40c8378de086f65f4554d95886897edae

I print the info line at the top of the CSV file and in a separate worksheet in 291b5d3dce9f0ed7c6856c07ba42fe39f4f2e4ca. I also removed the class `SistrCsvWriterShort` since it wasn't needed anymore (was used for testing before).

The only ISO formats I didn't switch are the excel formats since they didn't render properly by default in Excel. The code is also more complicated than it needs to be (if only I knew about pandas before I wrote all these code dealing with xlsxwriter).

Reply:
Apparently Python 3.5 does not support `timespec='seconds'`.  Removed in 8b420e85134133fc64bc2552bfa7f165ac25a1f7